src/fiatconnect/ReviewScreen.tsx

```diff
@@ -36,15 +42,35 @@ import { Currency } from 'src/utils/currencies'
 
 type Props = NativeStackScreenProps<StackParamList, Screens.FiatConnectReview>
 
+const disclaimerIsNeeded = (
+  flow: CICOFlow,
+  quote: FiatConnectQuote,
+  localCurrency: LocalCurrencyCode
+) => {
+  const fiatType = quote.getFiatType()
+  const currencyEligibleForDisclaimer =
+    !!CURRENCIES_WITH_FEE_DISCLAIMER[quote.getFiatAccountSchema()]?.has(localCurrency)
+  return (
+    flow === CICOFlow.CashOut &&
+    fiatType !== convertToFiatConnectFiatCurrency(localCurrency) &&
+    currencyEligibleForDisclaimer
+  )
+}
+
 export default function FiatConnectReviewScreen({ route, navigation }: Props) {
   const { t } = useTranslation()
   const dispatch = useDispatch()
   const { flow, normalizedQuote, fiatAccount, shouldRefetchQuote } = route.params
   const fiatConnectQuotesLoading = useSelector(fiatConnectQuotesLoadingSelector)
   const fiatConnectQuotesError = useSelector(fiatConnectQuotesErrorSelector)
+  const localCurrency = useSelector(getDefaultLocalCurrencyCode)
```

Comment:
nit: please call this something like `defaultLocalCurrency` b/c localCurrency is the variable name we use throughout the wallet for the currency that the user selects

Reply:
Removed that variable in a subsequent commit